components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -286,12 +294,14 @@ bool IsMediaLink(const GURL& url,
 
 // read comment about file pathes at src\base\files\file_path.h
 #if defined(OS_WIN)
+const base::FilePath::StringType kDiagnosticLogPath("rewards.log");
```

Comment:
`rewards` -> `Rewards`. Files in profile are mostly capitalized

Reply:
I know https://google.github.io/styleguide/cppguide.html#File_Names is for class names, but even file names I find this good practice to use lowercase

Reply:
everything in profile folder is capitalized (except of our files of course :) ). Let's try for new files be consistent  with that

Reply:
it is with the exception of `chrome_debug.log`

Reply:
yes, but that one is in root folder, not in Default folder

Reply:
Changed to `Rewards.log`